src/libYARP_OS/include/yarp/os/Random.h

```diff
@@ -7,6 +7,8 @@
 #ifndef YARP_OS_RANDOM_H
 #define YARP_OS_RANDOM_H
 
+#ifndef YARP_NO_DEPRECATED
```

Comment:
Please add
```diff
+#ifndef YARP_NO_DEPRECATED // Since YARP 2.3.72
```
So that we will find it easily when we will remove the deprecated features in the next release